pkgs/top-level/all-packages.nix

```diff
@@ -12418,7 +12418,9 @@ with pkgs;
 
   toxvpn = callPackage ../tools/networking/toxvpn { };
 
-  toybox = callPackage ../tools/misc/toybox { };
+  toybox = darwin.apple_sdk_11_0.callPackage ../tools/misc/toybox {
```

Comment:
Is this available on Linux as well?

Reply:
It's a no-op on Linux.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/platform-notes.chapter.md